source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/ResourceInformationTests.cpp

```diff
@@ -284,6 +289,14 @@ namespace AdaptiveCardsSharedModelUnitTest
                     {\
                         \"type\": \"CustomTypeWithImage\",\
                         \"customImageProperty\": \"Custom.png\"\
+                    },\
+                    {\
+                        \"type\": \"TextBlock\",\
+                        \"text\": \"this is a textblock\",\
+                        \"requires\": \
+                        {\
+                            \"CoolFeature\": \"1\"\
+                        }\
```

Comment:
Also this. Just wondering what we're testing here and whether it would be better suited to be in the ObjectModelTest.cpp file since this file is focused on resource information.

Reply:
actually -- good catch. this was a derp.

---
In reply to: [272648335](https://github.com/Microsoft/AdaptiveCards/pull/2623#discussion_r272648335) [](ancestors = 272648335)